src/cpp/session/CMakeLists.txt

```diff
@@ -41,21 +41,19 @@ if(NOT EXISTS "${RSTUDIO_DEPENDENCIES_DIR}/common/plumber")
   message(FATAL_ERROR "plumber package not found (re-run install-dependencies script to install)")
 endif()
 
-# verify libclang is installed
+# verify libclang is installed (Windows only)
 if(WIN32)
-   set(LIBCLANG_VERSION "3.4")
-else()
-   set(LIBCLANG_VERSION "3.5")
-endif()
-set(LIBCLANG_DIR "${RSTUDIO_DEPENDENCIES_DIR}/common/libclang/${LIBCLANG_VERSION}")
-if(NOT EXISTS "${LIBCLANG_DIR}")
-  message(FATAL_ERROR "libclang ${LIBCLANG_VERSION} not found  (re-run install-dependencies script to install)")
-endif()
-set(LIBCLANG_HEADERS_DIR "${RSTUDIO_DEPENDENCIES_DIR}/common/libclang/builtin-headers")
-if(NOT EXISTS "${LIBCLANG_HEADERS_DIR}")
-  message(FATAL_ERROR "libclang builtin-headers not found  (re-run install-dependencies script to install)")
-endif()
+   set(LIBCLANG_VERSION "5.0.2")
+   set(LIBCLANG_DIR "${RSTUDIO_DEPENDENCIES_DIR}/common/libclang/${LIBCLANG_VERSION}")
+   if(NOT EXISTS "${LIBCLANG_DIR}")
+      message(FATAL_ERROR "libclang ${LIBCLANG_VERSION} not found  (re-run install-dependencies script to install)")
+   endif()
 
+   set(LIBCLANG_HEADERS_DIR "${RSTUDIO_DEPENDENCIES_DIR}/common/libclang/builtin-headers")
```

Comment:
It looked to me like we are no longer downloading a `builtin-headers` directory, or are we?

Reply:
We aren't doing this anymore; instead, I just packaged everything together into a single zipfile. This includes 32bit and 64bit `libclang.dll` (including support libraries), and also the `libc++` sources.

Reply:
So this cmake code is still valid but points to the same core zip file, correct?

Reply:
Right -- effectively, unpacking `libclang-windows-5.0.2.zip` will ensure both the `libclang/5.0.2` and `libclang/builtin-headers/5.0.2` folders are both populated as they're now part of the same archive.